codalab/apps/teams/templates/teams/edit.html

```diff
@@ -46,7 +47,7 @@
     <h3><b>Team Information</b></h3>
     <div class="row">
         <div class="col-sm-12">
-            <table class="table table-striped affiliation">
+            <table id="teams-edit" class="table table-striped affiliation">
```

Comment:
Hmm, maybe you just made this change on this branch? I think that's fine, I merged in develop for you so as long as teams looks/behaves fine, I'm OK with this being in the PR :)